src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/IDebugBridge.cs

```diff
@@ -35,12 +35,17 @@ public interface IDebugBridge
         GethLikeTxTrace[] GetBlockTrace(Rlp blockRlp, CancellationToken cancellationToken, GethTraceOptions? gethTraceOptions = null);
         byte[] GetBlockRlp(Keccak blockHash);
         byte[] GetBlockRlp(long number);
+        Block? GetBlock(Keccak blockHash);
+        Block? GetBlock(long number);
```

Comment:
Why not `BlockParameter` and only one method?

Reply:
I didn't know this was possible, since I saw the two "GetBlockRlp" functions treat them separately.